internal/view/screen_dump.go

```diff
@@ -34,7 +35,7 @@ func NewScreenDump(gvr client.GVR) ResourceViewer {
 }
 
 func (s *ScreenDump) dirContext(ctx context.Context) context.Context {
-	dir := filepath.Join(s.App().Config.K9s.GetScreenDumpDir(), s.App().Config.K9s.CurrentCluster)
+	dir := filepath.Join(s.App().Config.K9s.GetScreenDumpDir(), dao.SanitizeFilename(s.App().Config.K9s.CurrentCluster))
```

Comment:
I think we should use CurrentContext here vs CurrentCluster as this is what a user is operating from.

Reply:
Also if you look at dao.ScreenDump the directory is sanitized twice now. I think your idea is correct but we should remove the sanitization of the dir in the dao.

Reply:
Good idea to normalize `CurrentContext`!
I've also changed the table saving to `CurrentContext` iso `CurrentCluster`.